docs/extending-overseerr/reverse-proxy-examples.md

```diff
@@ -123,3 +123,52 @@ Finally, reload `nginx` for the new configuration to take effect:
 ```bash
 sudo systemctl reload nginx
 ```
+
+## `nginx` (subfolder)
+
+{% hint style="warning" %}
+Nginx subfolder reverse proxy is unsupported. The sub filters may stop working when Overseerr is updated. Use at your own risk!
+{% endhint %}
+
+Add the following location block to your existing `nginx.conf` file.
+
+```nginx
+location ^~ /overseerr {
+    set $app 'overseerr';
+    # Remove /overseerr path to pass to the app
+    rewrite ^/overseerr/?(.*)$ /$1 break;
+    proxy_pass http://127.0.0.1:5055;  # NO TRAILING SLASH
+    # Redirect location headers
+    proxy_redirect ^ /$app;
+    proxy_redirect /setup /$app/setup;
+    proxy_redirect /login /$app/login;
+    # Sub filters to replace hardcoded paths
+    proxy_set_header Accept-Encoding "";
+    sub_filter_once off;
+    sub_filter_types *;
+    sub_filter 'href="/"' 'href="/$app"';
+    sub_filter 'href="/login"' 'href="/$app/login"';
+    sub_filter 'href:"/"' 'href:"/$app"';
+    sub_filter '/_next' '/$app/_next';
+    sub_filter '/api/v1' '/$app/api/v1';
+    sub_filter '/login/plex/loading' '/$app/login/plex/loading';
+    sub_filter '/images/' '/$app/images/';
+    sub_filter '/android-' '/$app/android-';
+    sub_filter '/apple-' '/$app/apple-';
+    sub_filter '/favicon' '/$app/favicon';
+    sub_filter '/logo.png' '/$app/logo.png';
+    sub_filter '/site.webmanifest' '/$app/site.webmanifest';
+}
+```
+
+Next, test the configuration:
+
+```bash
+sudo nginx -t
+```
+
+Finally, reload `nginx` for the new configuration to take effect:
+
+```bash
+sudo systemctl reload nginx
+```
```

Comment:
To keep this shorter and not to have duplicate text. You can use tabs. 

Example:
https://gist.github.com/samwiseg0/d920be8c3e8e84d0e2346d9dc76078f5